UPDATING.md

```diff
@@ -13,6 +13,20 @@ port socket driver, are also represented by a port and some matching code may ne
 `is_pid/1` to `is_port/1`.
 - Ports and pids can be registered. Function `globalcontext_get_registered_process` result now is
 a term that can be a `port()` or a `pid()`.
+- ESP32 builds with Elixir support may be configured without making changes to git tracked files
+using `idf.py -DATOMVM_ELIXIR_SUPPORT=on reconfigure` instead of copying partitions-elixir.csv to
```

Comment:
I would avoid the implicit esp32 target having been set and go straight for `idf.py -DATOMVM_ELIXIR_SUPPORT=on set-target ${CHIP}` - which works for all scenarios..

This also means a lot of the text can be deleted.